files/image_config/monit/container_checker

```diff
@@ -41,7 +66,15 @@ def get_expected_running_containers():
 
     expected_running_containers = set()
     always_running_containers = set()
-    
+
+    # Get current asic presence list. For multi_asic system, multi instance containers
+    # should be checked only for asics present.
+    asics_id_presence = get_asic_presence_list()
+
+    # Some services, like database and bgp run all the instances irrespective of asic presence.
+    # Add those to exception list.
```

Comment:
Lets remove this. We have to fix this properly not to even start this containers.

Reply:
do you mean both bgp and database? Should we keep this for now and remove it once that is fixed? I will raise a git issue to track the change. Let me know.

Reply:
added comments as discussed.